modules/run-consul/run-consul

```diff
@@ -530,6 +546,9 @@ function run {
       --enable-gossip-encryption)
         enable_gossip_encryption="true"
         ;;
+      --enable-connect)
+        enable_connect="true"
```

Comment:
Same here. Declare above as a `local` variable and pass it explicitly to other functions that need it.

Reply:
Takled in #202.